LICENSE.md

```diff
@@ -0,0 +1,194 @@
+Apache License
```

Comment:
Just regenerated the license file as an md with `usethis`, to get it to pick up in `DESCRIPTION`

Reply:
Sorry, was actually to resolve an R CMD check warning.